src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -671,11 +552,11 @@ <h4 class="text-center">
 
 
 
-    <div class="alert alert-warning" id="statusMessage">
+    <div class="alert alert-info" id="statusMessage">
           Total Logs gone through in last search: {*}<br /><input type="hidden" value="false" id="ifShowAll" /><input type="hidden" value="false" id="ifShowTestData" /></div>
 
                 
-            {*}
+            <br />
```

Comment:
Hmmmm it didn't happen to me (yet). I've seen this test passing and failing multiple times, even when the test fails, the comparison still returns something (almost) immediately. Can you try to see if it still happens?
Or it could be because of this misplaced `{*}`.